src/test/java/com/google/sps/CommentAnalysisTest.java

```diff
@@ -354,4 +355,11 @@ public void testOneWordMap() {
     expectedMap.put("nice", 1);
     Assert.assertEquals(expectedMap, singleComment.getWordFrequencyMap());
   }
+
+  @Test
+  public void testCommonWordParse() {
+    List<String> wordList = CommonWordsRetriever.getCommonWords();
+    int expectedWordCount = 22;
+    Assert.assertEquals(expectedWordCount, wordList.size());
```

Comment:
I would suggest adding more test instead of just doing wordList? just a suggestion:)